mltrace/entities/tests.py

```diff
@@ -17,11 +17,8 @@ def testZscore(self, df=pd.DataFrame, stdev_cutoff: float = 5.0):
         Checks to make sure there are no outliers using z score cutoff.
         """
         z_scores = (
```

Comment:
Let's delete this file as the custom tests are never actually used / we haven't started working on the library yet

Reply:
removed